src/posts/development_notam-2021-06-20.md

```diff
@@ -0,0 +1,109 @@
+---
+title: 'Development Updates June 2021'
+date: '2021-06-20'
```

Comment:
REMINDER: Update date upon ready to merge

Reply:
added to original comment to track. Title updated for now will change date for actual release.